wcag/2.5.3.md

```diff
@@ -2,4 +2,4 @@
 
 (Level A)
 
-Placeholder
+This applies directly as written, and as described in Intent from [Understanding Success Criterion 2.5.3](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html#intent).
```

Comment:
In WCAG2ICT, the links to understanding docs extends to `Intent from`, e.g. `[Intent from Understanding Success Criterion 2.5.3](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name.html#intent)`

If we agree to do the same, I can make adjustments in all files after merging this PR.